portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,49 +31,81 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+
+@WebServlet("/new-post")
 public class DataServlet extends HttpServlet {
  private ArrayList<String> check; 
```

Comment:
you can just initialize this as "private List<String> check = new ArrayList<>();"

Reply:
actually, this is probably fine (just realized that the initialization is in the init() method), but you can declare as "private List<String> check;" and later in the init() method do "check = new ArrayList<>();".

Reply:
I agree.